docs/source/modules/vert_coordinate.rst

```diff
@@ -0,0 +1,90 @@
+
++---------------------+--------------------------------------------------+
+| Name                | Description                                      |
++=====================+==================================================+
+|``vert_coord_option``| Sets model level scheme                          |
++---------------------+--------------------------------------------------+
+|``num_levels``       | Number of model levels                           |
++---------------------+--------------------------------------------------+
+|``scale_heights``    | Sets height of model top                         |
++---------------------+--------------------------------------------------+
+|``surf_res``         | Controls uniformity of model spacing             |
++---------------------+--------------------------------------------------+
+|``exponent``         | Controls bias of model levels towards troposphere|
++---------------------+--------------------------------------------------+
+|``p_press``          | Lower limit of pure pressure levels              |
++---------------------+--------------------------------------------------+
+|``p_sigma``          | Upper limit of pure sigma levels                 |
++---------------------+--------------------------------------------------+
+
+Prescribed Levels
+^^^^^^^^^^^^^^^^^
+Sometimes you may wish to manually set the pressure levels that are used in Isca: perhaps you want to directly compare a selection of similar models, or you wish to use the levels provided by another GCM. In order to use this mode we first set ``'vert_coord_option': 'input'``. In addition, we now need to supply a list of the levels we wish to use, which are added using the ``vert_coordinate`` namelist. The namelist ``bk`` contains the list of prescribed sigma levels that the model is to use - an example of which is shown below:
+
+.. code-block:: python
+
+  'vert_coordinate_nml': {
+        'bk': [0.000000, 0.040000, 0.100000, 0.200000, 0.400000, 0.600000, 0.800000, 1.000000],
+        'pk': [0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000, 0.000000],
+       }
+
+Even Sigma
+^^^^^^^^^^
+This option will create an evenly spaced set of levels from sigma level 0 (top) to 1 (surface), scaled by the surface pressure set in ``spectral_dynamics``. To use this scheme, set ``'vert_coord_option': 'even_sigma'`` and also provide a value for ``num_levels``. Increasing the number of levels will decrease the thickness of each model layer.
+
+Uneven Sigma
+^^^^^^^^^^^^
+The use of evenly-spaced model levels is often not the best option for modelling atmospheres, since it can lead to low resolution near the surface and in the troposphere, where the atmosphere is often the most dynamic). In addition, poor resolution near the model top can in certain configurations be unstable, since breaking gravity waves deposit significant amounts of energy into the upper layers. 
```

Comment:
I think there's a point to make in your first sentence here that it's more the other way around. If you split 1000hPa up into 10 levels, you'll have a 200hPa, 100hPa, and 0hPa level, meaning the entire stratosphere disappears into about 2 levels, whereas you'd get the other 8 in the troposphere.

Reply:
Yes agreed, probably got mixed up with vertical height z